libpod/container_commit.go

```diff
@@ -145,6 +144,5 @@ func (c *Container) Commit(ctx context.Context, destImage string, options Contai
 	if err != nil {
 		return nil, err
 	}
-	fmt.Fprintf(commitOptions.ReportWriter, importBuilder.Comment())
```

Comment:
Is this removing the display on the container's id after the commit completes?

Reply:
No printing in libpod